src/test/app/Oracle_test.cpp

```diff
@@ -666,6 +666,44 @@ struct Oracle_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testXRPAssetMeta(FeatureBitset features)
+    {
+        testcase("XRP Asset Meta");
+
+        Env env(*this, features);
+        Account const owner("owner");
+        env.fund(XRP(1'000), owner);
+        Oracle oracle(
+            env, {.owner = owner, .series = {{"XRP", "USD", 740, 1}}});
+
+        std::string const txHash{
+            env.tx()->getJson(JsonOptions::none)[jss::hash].asString()};
+
+        env.close();
+        Json::Value const meta = env.rpc("tx", txHash)[jss::result][jss::meta];
+
+        for (Json::Value const& metaNode : meta[sfAffectedNodes.jsonName])
+        {
+            // Ensure XRP asset is shown in PriceData regardless of the
+            // fixAMMMetadata amendment
```

Comment:
Why is this not amendment guarded?

Reply:
This is not a problem with Oracle I believe, since the asset is an inner object

Reply:
Why it should not be amendment guarded if it's an inner object?

Reply:
Inner object default values will not omitted. Metadata would only check the outer [object](https://github.com/XRPLF/rippled/blob/f6509495731b396ce33d96778e8df0366a8200bf/src/ripple/ledger/impl/ApplyStateTable.cpp#L236)